### PR TITLE
Add subscribe to YouTube message to README

### PR DESCRIPTION
**Pull request by mattseddon**
This PR adds a message in to the README asking users to subscribe to the YouTube channel for product updates. The README is currently shown in the GH repository and on the [marketplace listing](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc).

image

Please LMK if you would like to update/drop/move this message.

### CODE REVIEW

**Comment by mattseddon** [comment]
Also raised #4795 to remove the banner.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ Code. This extension uses [DVC](https://dvc.org/), an open-source data
 versioning and ML experiment management tool. No additional services or
 databases are required.
 
+🔔 Stay up-to-date with the latest product updates and tutorials by subscribing
+to the [DVC YouTube channel](https://www.youtube.com/@dvcorg8370/videos)!
+
+👉 [Subscribe now](https://www.youtube.com/@dvcorg8370/videos) to get notified
```
Comment (jendefig) [inline README.md:27]:
```suggestion
👉 [Subscribe to our YouTube channel](https://www.youtube.com/@dvcorg8370/videos) to get notified
```

Comment (mattseddon) [reply README.md:27]:
```
🔔 Don't miss out on the latest tips and tricks for using DVC in your projects!
Subscribe to the DVC YouTube channel now to stay up-to-date with the latest
product updates and tutorials.

👉 [Subscribe now](https://www.youtube.com/@dvcorg8370/videos) to get notified
whenever we post new content. You'll get access to exclusive tutorials, demos,
and best practices that will help you get the most out of DVC in your workflow.
Join our community of data scientists and developers today! 💻🚀
```
copilot also suggested this when I told it to "update the statement to show that subscribing will provide value"

Comment (jendefig) [reply README.md]: If we just leave with just "subscribe now" they may think it mean :
1. more email 🙅🏽‍♂️, or
2. I'm already subscribed

Clarifying that it's youtube will create less resistance.

**Review by jendefig — APPROVED** [review]
LGTM with the change.

File: README.md (hunk 1 above)
Comment (mattseddon) [reply README.md]: Do you think we can drop the second line? Seems repetitive.

Comment (mattseddon) [reply README.md]: or the first 🤷🏻

**Review by julieg18 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (julieg18) [reply README.md]: I'm also unsure about having two separate lines that are taking you to the same youtube page 🤔

Comment (jendefig) [reply README.md]: Actually, you are so right @julieg18! I missed that the first line had a link.  I think just the first line is fine.  Dump the second line I think.

**Pull request merged**